code/modules/clothing/masks/hailer.dm

```diff
@@ -1,10 +1,55 @@
+// **** Dispatch ****
+
+GLOBAL_LIST_EMPTY(sechailers)
+
+/datum/action/item_action/dispatch
+	name = "Signal dispatch"
+	desc = "Opens up a quick select wheel for reporting crimes, including your current location, to your fellow security officers."
+	button_icon_state = "dispatch"
+	icon_icon = 'icons/mob/actions/hailer_actions.dmi'
+
+/obj/item/clothing/mask/gas/sechailer
+	var/obj/item/radio/radio
+	var/radio_key = /obj/item/encryptionkey/headset_sec
+	var/radio_channel = RADIO_CHANNEL_SECURITY
+	COOLDOWN_DECLARE(dispatch_cooldown_timer)
+	var/dispatch_cooldown = 20 SECONDS
+
+/obj/item/clothing/mask/gas/sechailer/Destroy()
+	QDEL_NULL(radio)
+	GLOB.sechailers -= src
+	. = ..()
+
+/obj/item/clothing/mask/gas/sechailer/Initialize(mapload)
+	. = ..()
+	GLOB.sechailers += src
+	radio = new(src)
+	radio.keyslot = new radio_key
+	radio.listening = FALSE
+	radio.recalculateChannels()
+
+/obj/item/clothing/mask/gas/sechailer/proc/dispatch(mob/user)
+	if(COOLDOWN_TIMELEFT(src, dispatch_cooldown_timer))
+		to_chat(user, "<span class='notice'>Dispatch radio broadcasting systems are recharging.</span>")
+		return FALSE
+	var/list/options = list()
+	for(var/option in list("4 (capital crime)", "0 (infraction)", "1 (misdemeanor)", "2 (offense)", "3 (felony)")) //Hardcoded for each icon, not all crimes need emergency callout for more officers
+		options[option] = image(icon = 'icons/effects/aiming.dmi', icon_state = option)
+	var/message = show_radial_menu(user, user, options)
+	if(!message)
+		return FALSE
+	radio.talk_into(src, "Dispatch, code [message] in progress in [get_area(user)], requesting assistance.", radio_channel)
+	COOLDOWN_START(src, dispatch_cooldown_timer, dispatch_cooldown)
+	for(var/atom/movable/hailer in GLOB.sechailers)
+		if(ismob(hailer.loc))
+			playsound(hailer.loc, "sound/voice/sechailer/dispatch_please_respond.ogg", 100, FALSE)
```

Comment:
AI voice sounds like shit, and is way too long. Sounds like Miss Pauling lmao.

Really should just be a radio noise, and a muffled single word "dispatch" voiceline.

I'll trawl freesounds for somethin in the meantime